api/src/main/resources/ucum-fhir-essence.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+-->
+<root version="2.1" revision="$Revision: 442 $" revision-date="$Date: 2017-11-21 19:04:52 -0500 (Tue, 21 Nov 2017) $">
```

Comment:
this additional ucum-essence.xml is where we can define any UCUM definitions that aren't covered by the standard file provided by unitsofmeasure.org. currently we only define mmHg since it's used frequently in observation quantities (and the UCUM standard representation is mm[Hg])

Not a huge fan of this approach though since it feels like we're creating our own standardized library that now doesn't conform to UCUM definitions. I think maybe the better solution might be to extend ucum-fhir parsing to validate/parse via the <printSymbol> definition as well. but that project doesn't seem to be maintained regularly and this approach has a smaller barrier to entry so 🤷‍♂️ . Would be interested to hear opinions though